apps/storefront/components/Providers.jsx

```diff
@@ -24,10 +24,7 @@ export function Providers({ children, authApiURL }) {
   return (
     <>
       <LocalizationProvider dateAdapter={AdapterDateFns}>
-        <AuthProvider 
-          STORAGE_KEY={'customerAccessToken'}
```

Comment:
It's possible if we add it as a NEXT_PUBLIC env variable as it needs to be accessed on the client side.
We could use a normal env variable and pass it from the server side, but it would be too much of an overhead, and it wouldn't behave as an env variable (changing the value won't change it on the client side) unless I'm missing something.

With that said, I'd rather not do it at the time being as it requires a lot of unnecessary work :)